website/docs/sdk-reference/elixir.md

```diff
@@ -0,0 +1,251 @@
+| `key`           | **REQUIRED.** Setting-specific key. Set on *ConfigCat Dashboard* for each setting.                           |
+| `default_value` | **REQUIRED.** This value will be returned in case of an error.                                               |
+| `user`          | Optional, *ConfigCat.User Object*. Essential when using Targeting. [Read more about Targeting.](advanced/targeting.md) |
+```elixir
+value = ConfigCat.get_value(
+    "keyOfMySetting", # Setting Key
+    false, # Default value
+    ConfigCat.User.new("435170f4-8a8b-4b67-a723-505ac7cdea92") # Optional User Object
+);
+```
+
+### User Object
+The [User Object](../advanced/user-object.md) is essential if you'd like to use ConfigCat's [Targeting](advanced/targeting.md) feature.
+```elixir
+user_object = ConfigCat.User.new("435170f4-8a8b-4b67-a723-505ac7cdea92")
+```
+```elixir
+user_object = ConfigCat.User.new("john@example.com")
+```
+
+| Parameters   | Description                                                                                                                     |
+| ------------ | ------------------------------------------------------------------------------------------------------------------------------- |
+| `identifier` | **REQUIRED.** Unique identifier of a user in your application. Can be any value, even an email address.                         |
+| `email`      | Optional parameter for easier targeting rule definitions.                                                                       |
+| `country`    | Optional parameter for easier targeting rule definitions.                                                                       |
+| `custom`     | Optional `Map` for custom attributes of a user for advanced targeting rule definitions. e.g. User role, Subscription type. |
+```elixir
+user_object = ConfigCat.User.new("435170f4-8a8b-4b67-a723-505ac7cdea92", email: 'john@example', country: 'United Kingdom',
+                custom: ‰{SubscriptionType: 'Pro', UserRole: 'Admin'})
+```
+
+## Polling Modes
+The *ConfigCat SDK* supports 3 different polling mechanisms to acquire the setting values from *ConfigCat*. After latest setting values are downloaded, they are stored in the internal cache then all requests are served from there. With the following polling modes, you can customize the SDK to best fit to your application's lifecycle.
+
+### Auto polling (default)
+The *ConfigCat SDK* downloads the latest values and stores them automatically every 60 seconds.
+
+Use the `poll_interval_seconds` option parameter to change the polling interval.
+```elixir
+{ConfigCat, [
+    sdk_key: "YOUR SDK KEY",
+    cache_policy: ConfigCat.CachePolicy.auto(poll_interval_seconds: 60)
+]},
+```
+Adding a callback to `on_changed` option parameter will get you notified about changes.
+```elixir
+{ConfigCat, [
+    sdk_key: "YOUR SDK KEY",
+    cache_policy: ConfigCat.CachePolicy.Auto(on_changed: callback)
+]}
```

Comment:
Maybe mention a few other things here:
- `on_changed` needs to be a 0-arity function
- `on_changed` is run in a separate Process
- Any exceptions raised by `on_changed` are caught and logged

Reply:
Done 👍